packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.ts

```diff
@@ -100,10 +126,62 @@ export async function updateWalkthrough(context, lambdaToUpdate?: string) {
   const projectBackendDirPath = context.amplify.pathManager.getBackendDirPath();
   const resourceDirPath = path.join(projectBackendDirPath, category, functionParameters.resourceName);
   const currentParameters = loadFunctionParameters(context, resourceDirPath);
+  const functionRuntime = context.amplify.readBreadcrumbs(context, category, functionParameters.resourceName).functionRuntime as string;
+
+  // Provide general information
+  context.print.success('General information');
+  context.print.info('| Name: '.concat(lambdaToUpdate));
+  context.print.info('| Runtime: '.concat(functionRuntime));
+  context.print.info('');
+
+  // Provide resource access permission information
+  context.print.success('Resource access permission');
+  const currentCategoryPermissions = fetchPermissionCategories(currentParameters.permissions);
+  if (currentCategoryPermissions.length) {
+    currentCategoryPermissions.forEach(category => {
+      const currentResources = fetchPermissionResourcesForCategory(currentParameters.permissions, category);
+      currentResources.forEach(resource => {
+        const currentPermissions = fetchPermissionsForResourceInCategory(currentParameters.permissions, category, resource);
+        const formattedCurrentPermissions = ' ('.concat(currentPermissions.join(', ').concat(')'));
+        context.print.info('- '.concat(resource).concat(formattedCurrentPermissions));
+      });
+    });
+  } else {
+    context.print.info('- Not configured');
+  }
+  context.print.info('');
 
-  if (
-    await context.amplify.confirmPrompt('Do you want to update the Lambda function permissions to access other resources in this project?')
-  ) {
+  // Provide scheduling information
+  const cfnParameters = context.amplify.readJsonFile(path.join(resourceDirPath, parametersFileName), undefined, false) || {};
+  const scheduleParameters = {
+    cloudwatchRule: cfnParameters.CloudWatchRule,
+    resourceName: functionParameters.resourceName,
+  };
+  context.print.success('Scheduled recurring invocation');
+  if (scheduleParameters.cloudwatchRule && scheduleParameters.cloudwatchRule !== 'NONE') {
+    context.print.info('| '.concat(scheduleParameters.cloudwatchRule));
+    context.print.info('');
+  } else {
+    context.print.info('| Not configured');
+    context.print.info('');
+  }
+
+  // Provide lambda layer information
+  context.print.success('Lambda layers');
+  if (currentParameters.lambdaLayers.length) {
```

Comment:
Check if current Parameters object contain lambdalayers
```javascript
if (currentParameters.lambdaLayers?.length) 
```

Reply:
Added a check that currentParameters contains lambdaLayers